src/aerie_cli/aerie_host.py

```diff
@@ -40,10 +40,10 @@ def __init__(self, encoded_jwt: str) -> None:
         encoded_jwt_payload = b64decode(jwt_components[1] + "==", validate=False)
         try:
             payload = json.loads(encoded_jwt_payload)
-            self.active_role = payload["activeRole"]
```

Comment:
Why not set the `active_role` to the `default_role` here?

Reply:
I just separated the notion of an active role from the `AerieJWT` object since it's no longer part of the JWT.